### ISSUE TITLE
Passing boolean columns

### ISSUE THREAD

**Issue by pfk-beta** [jdb78/pytorch-forecasting#480]
Hello,

I have couple of columns in my dataset, like "is_summer", "is_holiday", which are `0` or `1`. Should I convert them to `str` and treat them as `time_varying_known_categoricals`.
I have also column with `week_number` and `year`, is it ok that `year` is in `time_varying_known_reals` and `week_number` in `time_varying_known_categoricals`

**Comment by jdb78** [comment]
Yes, this would work. I would convert them to strings and add them to the categoricals. You can put categorical variables into [`variable_groups`](https://pytorch-forecasting.readthedocs.io/en/latest/data.html#time-series-data-set) if they are similar, e.g. related to time.

**Comment by pfk-beta** [comment]
something like this:

```Python
TimeSeriesDataSet(
    . . .
    time_varying_known_categoricals=['is_january', 'is_december', 'is_summer', 'is_friday', 'is_monday', 'week_holiday', 'day_of_week', 'week_number',]
    variable_groups={
        'month_group': ['is_january', 'is_december', 'is_summer',],
        'week_group': ['is_friday', 'is_monday', 'week_holiday', 'day_of_week', 'week_number',]
    }
    . . .
)
```

**Comment by jdb78** [comment]
You can absolutely do this. However, I think if you have non-overlapping categoricals, I would put them into one variable (e.g. day of the week or month).
